cloudhub-2/modules/ROOT/pages/ps-assign-permissions.adoc

```diff
@@ -6,12 +6,16 @@ assigned to your user account or a team to which you belong.
 To assign permissions, you need access to Access Management and the Organization Administrator permission.
 
 * To set up and manage a private space, you need the CloudHub Network Administrator permission.
-* To view a private space, you need the CloudHub Network Viewer permission.
-** To view the schedules in applications deployed to CloudHub 2.0 and Runtime Fabric, you must assign the Exchange Viewer permission to the Business Organization that owns the asset.
+* To view a private space as an individual user within an organization, you need at least one of the following permissions:
+** Cloudhub Network Viewer
+** Organization Administrator
```

Comment:
And also add a line on what happens if they don't have these permissions, then they will see this screen below.
![Screenshot 2023-11-14 at 12 58 54 PM](https://github.com/mulesoft/docs-hosting/assets/22915931/01b6063d-7eea-465a-a417-c97ca1c88c78)